source/GameAction.cpp

```diff
@@ -499,11 +514,14 @@ GameAction GameAction::Instantiate(map<string, string> &subs, int jumps, int pay
 
 	result.debt = debt;
 
-	if(!logText.empty())
-		result.logText = Format::Replace(logText, subs);
-	for(auto &&it : specialLogText)
+	result.logMedia = logMedia;
+	for(auto &media : result.logMedia)
+		media.FormatReplace(subs);
```

Comment:
Why not create an `Instantiate` method that takes the subs and returns a new object with the subs applied instead of this?